src/swagger-specs/batch-ingestion.yaml

```diff
@@ -0,0 +1,1200 @@
+openapi: 3.0.1
+info:
+  title: Data Ingestion API
+  description: |-
+    Data Ingestion allows you to bring your data into Adobe Experience
+    Platform through batch ingestion and streaming ingestion. Batch ingestion lets
+    you import data in batch, from any number of data sources. Streaming ingestion
+    lets users send data to Platform in real time from client and server-side devices.
```

Comment:
Description for Batch can be updated to something like: "Batch ingestion allows you to ingest data into Adobe Experience Platform as batch files. Batches are units of data that consist of one or more files to be ingested as a single unit. Once ingested, batches provide metadata that describes the number of records successfully ingested, as well as any failed records and associated error messages."